### ISSUE TITLE
How to change locale of numerical values?

### ISSUE THREAD

**Issue by hamzabinamin** [AAChartKit/AAChartKit-Swift#77]
@AAChartModel @HuangYanQi  I understand that HighChart displays numerical values as 1 092 instead of 1,092. Is there a way I can change the locale to display "," instead of space?

**Comment by hamzabinamin** [comment]
@Danixfun

**Comment by AAChartModel** [comment]
Change the content of AAChartView.html

```js
                aaOptions.credits = {
                    enabled: false
                }; //去掉表格右下角版权信息
                
```

to be

```js
       aaOptions.credits = {
                    enabled: false
                }; //去掉表格右下角版权信息
                
                //全局配置(可通过全局配置设置主题)https://api.hcharts.cn/highcharts#Highcharts.setOptions
                Highcharts.setOptions({
                                      lang: {
                                          thousandsSep: ","
                                      }
                                      });
```

It means that just add the content 

```js
                Highcharts.setOptions({
                                      lang: {
                                          thousandsSep: ","
                                      }
                                      });
```

into the AAChartView.html

**Comment by AAChartModel** [comment]
Refer to the link:   https://api.highcharts.com/highcharts/lang.thousandsSep

**Comment by AAChartModel** [comment]
@hamzabinamin  @Danixfun   I fixed it in  d7a80ac , but I do not sync it with cocoapods , so you can change it by yourself .😊
